src/style.css

```diff
@@ -1 +1,79 @@
 /* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+}
+
+.nav {
+  position: relative;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav__link {
+  cursor: pointer;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  padding: 23px 0;
+  margin-left: 20px;
+  height: 15px;
+  width: 37px;
+}
+
+.nav__link:hover {
+  text-decoration: underline;
+  text-decoration-color: #00acdc;
+  text-decoration-thickness: 2px;
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+  text-decoration: none;
+  padding: 23px 0;
+  height: 15px;
+  width: 37px;
```

Comment:
Don't fix this values for the link itself
the height and the width should depend on content
and the padding is not required because you just center the text vertically